### ISSUE TITLE
Headings not being set on QueuedExport

### ISSUE THREAD

**Issue by hotmeteor** [Maatwebsite/Laravel-Nova-Excel#15]
I've set up a `QueuedExport` and am using `->withHeadings()` but headings are not being prepended to my export.

```php
(new QueuedExport())
	->withHeadings('ID', 'First Name', 'Last Name', 'Email', 'Phone', 'City', 'State', 'Zip', 'Status')
	->only('id', 'first_name', 'last_name', 'email', 'phone', 'city', 'region', 'postal_code', 'worker_status')
	->withDisk(config('filesystems.default'))
	->withFilename($export_filename)
	->onSuccess(function (ActionRequest $request, PendingDispatch $queue) use ($export_filename) {
		$queue
			->allOnQueue('default')
			->chain([
				new NotifyUserOfCompletedExport(
					$request->user(),
					$export_filename,
					Storage::temporaryUrl($export_filename, Carbon::now()->addWeek())
				),
			]);

			return Action::message('Your export is queued. You will be notified when it\'s ready.');
		}),
```

**Comment by patrickbrouwers** [comment]
@hotmeteor Fixed in 1.0.3! 👍

**Comment by hotmeteor** [comment]
Legend! Thanks!
